test/it/ChoiceSpec.scala

```diff
@@ -0,0 +1,48 @@
+import forms.Choice
+import models.cache.{Cache, DepartureAnswers}
+import play.api.http.Status
```

Comment:
You don't need to import `Status`. All statuses are in `play.api.tet.Helpers._` which you imported :)